contributing-device-support.md

```diff
@@ -3,7 +3,7 @@
 There are several ways for enabling balenaOS support for your hardware:
 
 1. Checking out if balenaOS doesn't already provide support for your board through our [supported devices list](https://hub.balena.io/device-types).
-2. Consider our [Custom Device Support](https://www.balena.io/pricing/#custom-device-support) (CDS) service. This is a paid service where we create a custom balenaOS build for you and maintain it for a monthly fee. Check out out [our overview](https://drive.google.com/file/d/17-DTeXkflkzW8Qduj48ut7JNUTWqiRv-/view?usp=sharing) to learn more.
+2. Consider our [Custom Device Support (CDS) service](https://custom-device-support.pages.dev/). This is a paid service where we create a custom balenaOS build for you and maintain it for a monthly fee.
```

Comment:
hey @rhampt what's the story behind the `pages.dev` domain? Why not a `balena-io` subdomain?

Reply:
It's all in the [CDS Productization Zulip topic ](https://balena.zulipchat.com/#narrow/stream/345889-loop.2Fbalena-os/topic/CDS.20productization)@alexgg . TL;DR - That's the static site URL that @anujdeshpande was able to generate w/ Flowzone.